docs/guides/06-errors.md

```diff
@@ -186,19 +186,25 @@ curl http://localhost:4010/todos -H "accept: application/json"`
 
 **Returned Status Code: `404`**
 
-**Explanation:** This error occurs when the current request is asking for a specific status code that the document isn't listing or it's asking for a specific example that doesn't exist in the current document
+**Explanation:** This error occurs when the current request is asking for a specific status code that the document isn't listing or it's asking for a specific example that doesn't exist in the current document.
 
 ### VIOLATIONS
 
-**Message: Request/Response not valid**
+**Message #1: Request/Response not valid**
 
 **Returned Status Code: `500`**
 
-**Explanation:** This error occurs when you're run Prism with the `--errors` flag and the request or the response has at least one violation marked as an error
+**Explanation:** This error occurs when you're run Prism with the `--errors` flag and the request or the response has at least one violation marked as an error.
+
+**Message #2: response.body Request body must match exactly one schema in oneOf**
+
+**Return Status Code: N/A**
+
+**Explanation:** This error can occur if the current document contains a discriminator. Prism does not support discriminators at this time. 
```

Comment:
I think this description is clear.  Entirely at your option, you might consider linking the word "discriminator" to some OpenAPI documentation version, e.g. https://spec.openapis.org/oas/v3.0.1.html#discriminator-object.